client/packages/host/src/Host.tsx

```diff
@@ -58,6 +58,11 @@ Bugsnag.start({
 const skipRequest = () =>
   LocalStorage.getItem('/auth/error') === AuthError.NoStoreAssigned;
 
+const PluginProvider: React.FC<PropsWithChildren> = ({ children }) => {
```

Comment:
this is required because the plugins will need to access the various providers in the hierarchy. I've had to call the hook at the right level